specification/search/data-plane/Microsoft.Azure.Search.Data/preview/2017-11-11-preview/searchindex.json

```diff
@@ -64,6 +64,325 @@
         }
       }
     },
+    "/docs": {
+      "get": {
+        "tags": [
+          "Search"
+        ],
+        "operationId": "DocumentsProxy_SearchGet",
+        "x-ms-examples": {
+          "SearchIndexGetSearch": { "$ref": "./examples/SearchIndexGetSearch.json" }
+        },        
+        "description": "Searches for documents in the Azure Search index.",
+        "parameters": [
+          {
+            "name": "search",
+            "in": "query",
+            "type": "string",
+            "description": "A full-text search query expression; Use null or \"*\" to match all documents.",
+            "x-ms-client-name": "searchText"
+          },
+          {
+            "name":"$count",
+            "in": "query",
+            "type": "boolean",
+            "description": "A value that specifies whether to fetch the total count of results. Default is false. Setting this value to true may have a performance impact. Note that the count returned is an approximation.",
+            "x-ms-client-name": "IncludeTotalResultCount",
+            "x-ms-parameter-grouping": {
+              "name": "SearchParameters"
+            }
+          },
+          {
+            "name":"facet",
+            "in": "query",
+            "type": "array",
+            "items": {
+              "type": "string"
+            },
+            "collectionFormat": "multi",
+            "description": "The list of facet expressions to apply to the search query. Each facet expression contains a field name, optionally followed by a comma-separated list of name:value pairs.",
+            "x-ms-client-name": "Facets",
+            "x-ms-parameter-grouping": {
+              "name": "SearchParameters"
+            }
+          },
+          {
+            "name":"$filter",
+            "in": "query",
+            "type": "string",
+            "description": "The OData $filter expression to apply to the search query.",
+            "x-ms-client-name": "Filter",
+            "x-ms-parameter-grouping": {
+              "name": "SearchParameters"
+            }
+          },
+          {
+            "name":"highlight",
+            "in": "query",
+            "type": "array",
```

Comment:
Do we need `collectionFormat: csv` here?

Reply:
csv is the default. [https://swagger.io/docs/specification/2-0/describing-parameters/](https://swagger.io/docs/specification/2-0/describing-parameters/) 

---
In reply to: [208101778](https://github.com/Azure/azure-rest-api-specs/pull/3578#discussion_r208101778) [](ancestors = 208101778)